Clients/src/presentation/components/Table/index.tsx

```
@@ -157,65 +168,111 @@ const BasicTable = ({
+                    <TableCell>{row.vendor_name}</TableCell>
+                    <TableCell>{row.risk_name}</TableCell>
+                    <TableCell>{row.owner}</TableCell>
+                    <TableCell>{row.risk_level}</TableCell>
+                    <TableCell>{row.review_date
+                      ? formatDate(row.review_date.toString())
+                      : "No review date"}
+                    </TableCell>
+                  </>}  
+                </TableRow>
+              </>}
+            </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)